apis/ec2/v1alpha2/referencers.go

```diff
@@ -14,7 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package v1alpha1
+package v1alpha2
```

Comment:
@haarchri you can just name this package and directory something like `manualv1alpha1` and then the generated one can use `v1alpha1`